### PR TITLE
Added exchange.go 

### PR DESCRIPTION
**Pull request by issacboil**
This command was made by Shadow and Mr. Ben, but shadow doesn't know how to use git, so I am pull requesting it for him. 

Co-Authored-By: Shadow <66721908+Shadow21AR@users.noreply.github.com>

### CODE REVIEW

File: stdcommands/exchange/exchange.go [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+
+	"github.com/botlabs-gg/yagpdb/v2/commands"
+	"github.com/botlabs-gg/yagpdb/v2/lib/dcmd"
+	"github.com/botlabs-gg/yagpdb/v2/lib/discordgo"
+)
+
+var Command = &commands.YAGCommand{
+	CmdCategory:         commands.CategoryFun,
+	Cooldown:            5,
+	Name:                "exchange",
```
Comment (mrbentarikau) [inline stdcommands/exchange/exchange.go:18]: Command name shoult start with capital letter.

File: stdcommands/exchange/exchange.go [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+		to := check.Symbols[strings.ToUpper(data.Args[2].Str())]
+		if (to == nil) || (from == nil) {
+			return "Invalid currency code.\nCheck out available codes on: <https://api.exchangerate.host/symbols>", nil
+		}
+		output, err := requestAPI("https://api.exchangerate.host/convert?from=" + from.Code + "&to=" + to.Code + "&amount=" + amount)
+		if err != nil {
+			return nil, err
+		}
+		embed := &discordgo.MessageEmbed{
+			Title:       "💱Currency Exhange Rate",
```
Comment (mrbentarikau) [inline stdcommands/exchange/exchange.go:45]: Typo

**Review by mrbentarikau — CHANGES_REQUESTED** [review]

**Comment by ashishjh-bst** [comment]
Please add screenshots of sample command usage.

**Comment by issacboil** [comment]
image`

Here is a picture of the command usage.

File: stdcommands/exchange/exchange.go [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+		output, err := requestAPI("https://api.exchangerate.host/convert?from=" + from.Code + "&to=" + to.Code + "&amount=" + amount)
+		if err != nil {
+			return nil, err
+		}
+		embed := &discordgo.MessageEmbed{
+			Title:       "💱Currency Exchange Rate",
+			Description: fmt.Sprintf("\n%s **%s** (%s) is %0.2f **%s** (%s).", amount, from.Description, output.Query.From, output.Result, to.Description, output.Query.To),
+			Color:       0xAE27FF,
+			Footer:      &discordgo.MessageEmbedFooter{Text: fmt.Sprintf("Based on currency rate 1 : %0.3f", output.Result)},
+            Timestamp:   time.Now().UTC().Format(time.RFC3339),
```
Comment (ashishjh-bst) [inline stdcommands/exchange/exchange.go:50]: formatting

File: stdcommands/exchange/exchange.go [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+	}
+	result := &Result{}
+	err = json.Unmarshal([]byte(body), &result)
+	if err != nil {
+		return nil, err
+	}
+	return result, nil
+}
+
+type SymbolInfo struct {
```
Comment (ashishjh-bst) [inline stdcommands/exchange/exchange.go:82]: Change name to CurrencySymbolInfo?

**Pull request merged**